C/CompilerSupportLibraries/common.jl

```diff
@@ -113,6 +113,10 @@ if [[ ${target} == *apple* ]]; then
     done
 fi
 
+for lib in libgfortran libasan libtsan libubsan liblsan; do
+  [ ! -f "${libdir}/${lib}.so" ] || patchelf --set-rpath "\$ORIGIN" "${libdir}/${lib}.so"
+done
+
```

Comment:
See https://github.com/JuliaPackaging/Yggdrasil/issues/3703#issuecomment-1291225562.  I believe this can be fixed in the audit pass, instead of having to manually fix it here.

Reply:
that would be somewhat simpler, but @staticfloat had thought it might fail in the verifier, so he suggested we do this manually

Reply:
I replied in https://github.com/JuliaPackaging/Yggdrasil/issues/3703#issuecomment-1291264529, I think the assessment was ill-informed